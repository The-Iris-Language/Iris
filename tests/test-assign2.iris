.. Test declaring and assigning local string variable in Main 
class Main () {
    public: 
        int univ main() {
            string hello;
            hello = "hello";
            Olympus.println("Variable Assigned!");
            return 0;
        }
}