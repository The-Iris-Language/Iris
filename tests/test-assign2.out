Hello this is a variable!